### DISCUSSION TITLE
Vim motions on tui

### DISCUSSION THREAD

**Discussion by AlvaroParker** [vercel/turborepo#9227] [Ideas]
### Goals

1. Add simple `j` and `k` vim motions on the tui to go down and up. 

### Background

Currently this is only implemented using the arrow keys. It would be nice to have  a way to configure vim motions on the tui. 

### Proposal

on the `turborepo-ui` crate in the `src/tui/input.rs` file this can probably be added by just adding another element on the match statement. I can work on it if this is approved.

**Comment by dimitropoulos** [comment]
We would happily accept a contribution for this!  Since releasing TUI we've seen some great interest in it (including people asking for custom themes, custom keybindings your question here, ways to resize/reorganize it, etc.) and that's made us develop interest in a [query interface](https://github.com/vercel/turborepo/discussions/9170).

**Reply by AlvaroParker** [reply]
@dimitropoulos I went ahead and did a PR if you want to check it out #9243

**Reply by anthonyshew** [reply]
Merged!
